### PR TITLE
Order by sequence

### PR DESCRIPTION
**Pull request by Anbranin**
screen shot 2018-06-22 at 3 20 19 pm

### CODE REVIEW

File: app/views/bus_stops/by_sequence.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.page-title Route #{@route.number} Bus Stops
+
+.container
+  - @collection.each do |direction, bsrs|
+    = direction + 'bound'
```
Comment (mboneil10) [inline app/views/bus_stops/by_sequence.haml]: I called `BusStopsRoute.pluck(:direction).uniq` and got `["South", "North", "West", "East", "Downward", "Upward"]`. Maybe "downwardbound" looks strange, but it doesn't really matter.

File: app/views/bus_stops/index.haml [hunk 2]
```diff
@@ -10,7 +10,11 @@ or
...
   = submit_tag 'Search'
 or
-= form_tag by_route_bus_stops_path, method: :get do
+= form_tag by_status_bus_stops_path, method: :get do
   = label_tag :number, 'Select a route'
   = select_tag :number, options_for_select(Route.pluck :number)
-  = submit_tag 'Search'
+  = submit_tag 'View stops by status'
+= form_tag by_sequence_bus_stops_path, method: :get do
+  = label_tag :number, 'Select a route'
```
Comment (mboneil10) [inline app/views/bus_stops/index.haml]: Idk how I feel about the label that is repeated.

File: app/views/bus_stops/by_sequence.haml (hunk 1 above)
Comment (dfaulken) [reply app/views/bus_stops/by_sequence.haml]: Do we need the 'bound'?

Comment (mboneil10) [reply app/views/bus_stops/by_sequence.haml]: No. Let's go without it.

**Pull request merged**